builder/util/util.go

```diff
@@ -0,0 +1,83 @@
+package util
```

Comment:
I didn't want to go through the headache of getting the `"github.com/docker/docker/builder"` package to compile for the client binary distribution on all platforms. It's easier to put it in a separate package.